network/dag/dag.go

```diff
@@ -96,69 +96,6 @@ func newDAG(db stoabs.KVStore) *dag {
 	return &dag{db: db}
 }
 
-func (d *dag) Migrate() error {
-	return d.db.Write(context.Background(), func(tx stoabs.WriteTx) error {
-		writer := tx.GetShelfWriter(metadataShelf)
-		// Migrate highest LC value
-		// Todo: remove after V5 release
-		_, err := writer.Get(stoabs.BytesKey(highestClockValue))
-		if errors.Is(err, stoabs.ErrKeyNotFound) {
-			log.Logger().Info("Highest LC value not stored, migrating...")
-			highestLC := d.getHighestClockLegacy(tx)
-			err = d.setHighestClockValue(tx, highestLC)
-		}
-		if err != nil {
-			return err
-		}
-
-		// Migrate number of TXs
-		// Todo: remove after V5 release
-		_, err = writer.Get(stoabs.BytesKey(numberOfTransactionsKey))
-		if errors.Is(err, stoabs.ErrKeyNotFound) {
-			log.Logger().Info("Number of transactions not stored, migrating...")
-			numberOfTXs := d.getNumberOfTransactionsLegacy(tx)
-			err = d.setNumberOfTransactions(tx, numberOfTXs)
-		}
-		if err != nil {
-			return err
-		}
-
-		// Migrate headsLegacy to single head
-		// Todo: remove after V6 release => then remove headsShelf
```

Comment:
I'm assuming the second half of this todo points to a now unused constant that should be removed?

Reply:
unused shelf, but since the entire DB will be removed in the future...